jmix-dynattr/dynattr/src/main/java/io/jmix/dynattr/model/CategoryAttribute.java

```diff
@@ -95,6 +97,8 @@ public class CategoryAttribute implements Serializable {
     @Column(name = "NAME", nullable = false)
     private String name;
 
+    @NotNull(message = "{msg://io.jmix.dynattr.model/CategoryAttribute.code.validation.NotNull}")
+    @Pattern(message = "{msg://io.jmix.dynattr.model/CategoryAttribute.code.validation.wrongCodeValueFormat}", regexp = "^[a-z]\\w*$")
```

Comment:
message already has `code` in this name, so no need to duplicate it. `msg://io.jmix.dynattr.model/CategoryAttribute.code.validation.wrongValueFormat` is OK.

Reply:
Agree. Fixed